heron/statemgrs/src/java/com/twitter/heron/statemgr/FileSystemStateManager.java

```diff
@@ -182,6 +183,13 @@ public Lock getLock(String topologyName, LockName lockName) {
   }
 
   @Override
+  public ListenableFuture<TopologyMaster.MetricsCacheLocation> getMetricsCacheLocation(
```

Comment:
There's a method somewhere under state manager where we delete all FS data associated with a topology. Please update to include this new data.